### PR TITLE
Remove helper `isTheme`

### PR DESCRIPTION
**Pull request by piotrzarycki**
remove helper isTheme #8854

### CODE REVIEW

File: src/amo/components/AddonAdminLinks/index.js [hunk 1]
```diff
@@ -48,10 +48,12 @@ export class AddonAdminLinksBase extends React.Component<InternalProps> {
...
 
-    const showCodeReviewLink = hasCodeReviewPermission && !isTheme(addon.type);
-    const showThemeReviewLink = hasThemeReviewPermission && isTheme(addon.type);
+    const showCodeReviewLink =
+      hasCodeReviewPermission && ADDON_TYPE_STATIC_THEME !== addon.type;
+    const showThemeReviewLink =
+      hasThemeReviewPermission && ADDON_TYPE_STATIC_THEME === addon.type;
     const showContentReviewLink =
-      hasContentReviewPermission && !isTheme(addon.type);
+      hasContentReviewPermission && ADDON_TYPE_STATIC_THEME !== addon.type;
```
Comment (willdurand) [inline src/amo/components/AddonAdminLinks/index.js]: How about introducing a `isTheme` variable above and use it?

```js
const isTheme = addon.type === ADDON_TYPE_STATIC_THEME;

const showThemeReviewLink = hasThemeReviewPermission && isTheme;
// ...
```

Comment (piotrzarycki) [reply src/amo/components/AddonAdminLinks/index.js]: good idea :), fixed

File: src/amo/components/AddonAdminLinks/index.js [hunk 2]
```diff
@@ -48,10 +48,15 @@ export class AddonAdminLinksBase extends React.Component<InternalProps> {
       return null;
     }
 
-    const showCodeReviewLink = hasCodeReviewPermission && !isTheme(addon.type);
-    const showThemeReviewLink = hasThemeReviewPermission && isTheme(addon.type);
+    const isTheme = addon.type === ADDON_TYPE_STATIC_THEME;
+    const isNotTheme = addon.type !== ADDON_TYPE_STATIC_THEME;
```
Comment (willdurand) [inline src/amo/components/AddonAdminLinks/index.js]: I think we do not need this variable. Use `!isTheme` instead.

Comment (piotrzarycki) [reply src/amo/components/AddonAdminLinks/index.js]: yees ;p clean code level hard ;p

File: src/amo/components/AddonAdminLinks/index.js [hunk 3]
```diff
@@ -48,10 +48,14 @@ export class AddonAdminLinksBase extends React.Component<InternalProps> {
...
     }
 
-    const showCodeReviewLink = hasCodeReviewPermission && !isTheme(addon.type);
-    const showThemeReviewLink = hasThemeReviewPermission && isTheme(addon.type);
+    const isTheme = addon.type === ADDON_TYPE_STATIC_THEME;
+
+    const showCodeReviewLink =
+      hasCodeReviewPermission && !isTheme;
+    const showThemeReviewLink =
+      hasThemeReviewPermission && isTheme;
```
Comment (willdurand) [inline src/amo/components/AddonAdminLinks/index.js]: Could you please reformat the lines? I am not sure running `yarn prettier-dev` will be enough.

Comment (piotrzarycki) [reply src/amo/components/AddonAdminLinks/index.js]: pls check

**Comment by willdurand** [comment]
Thanks!

**Comment by caitmuenster** [comment]
Thanks so much, @piotrzarycki! ✨ Your contribution has been added to our [recognition wiki](ttps://wiki.mozilla.org/Add-ons/Contribute/Recognition). 

Would you be interested in creating a profile on [mozillians.org](https://mozillians.org/)? I'd be happy to vouch for you!

**Comment by piotrzarycki** [comment]
@caitmuenster 
Thank you :). Sure, here is my profile https://mozillians.org/pl/u/piotr.zarycki/

**Comment by caitmuenster** [comment]
Awesome! You're all vouched, @piotrzarycki. :) Welcome onboard! I look forward to seeing you around the project.

**Comment by piotrzarycki** [comment]
Thx :)

**Pull request merged**